docs/extended-tour.md

```diff
@@ -21,7 +21,7 @@ This skeleton includes a few examples of how you might design your application,
 
 
 ### When would you use a standalone function?
```

Comment:
Just realizing that this docs/ directory should be deleted since it is a copy/paste from node-cpp-skel and does not apply to vtshaver directly.

Reply:
:+1: done in 595ade824353e7e6169e24c8155ee63ad50d1802